docs/contributing.md

```diff
@@ -0,0 +1,22 @@
+# Looking to contribute to the project?
+ see below for links and information related to your intended contributions
+
+## For raising issues and reporting bugs see the links below for further information
+- ###  [Bug reporting](https://github.com/SOFTENG701G1/A1/wiki/Bug-Reporting)  
```

Comment:
Save yourself some work and just remove the bug reporting page, issues can cover bugs :)